<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrowsePic</class>
 <widget class="QWidget" name="BrowsePic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BrowsePic</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="9,1,1" columnstretch="1,1,1,1,1,1">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0" colspan="6">
    <widget class="CMyLabel" name="label_show">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="btn_prePic">
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btn_openPic">
     <property name="text">
      <string>打开图像</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="btn_nextPic">
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QCheckBox" name="checkBox_equalScale">
     <property name="text">
      <string>等比缩放</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QCheckBox" name="checkBox_showCenter">
     <property name="text">
      <string>居中显示</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_status">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="btn_resetPic">
     <property name="text">
      <string>恢复</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CMyLabel</class>
   <extends>QLabel</extends>
   <header location="global">cmylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
